packages/cf-component-icon/example/basic/component.js

```diff
@@ -3,8 +3,15 @@ import Icon from 'cf-component-icon';
 
 const IconComponent = () => (
   <div>
-    <p>Types:</p>
-    <p>
+    <p>Types (hover for type prop value):</p>
+    <div
+      style={{
+        display: 'grid',
```

Comment:
I want to clarify, should we promote use of grid layout with IE/Edge support old format and no support in Safari 9 https://caniuse.com/#search=grid?